projects/novo-elements/src/elements/data-table/data-table-clear-button.component.ts

```diff
@@ -44,6 +44,8 @@ export class NovoDataTableClearButton<T> {
   queryClear: EventEmitter<boolean> = new EventEmitter();
   @Output()
   allClear: EventEmitter<boolean> = new EventEmitter();
+  @Input()
+  skipUpdate: boolean;
```

Comment:
we might only be concerned about the clearSearch function for the work we're doing now, but this input will be confusing if it's only applied to that one function and none of the others.